README.md

```diff
@@ -63,13 +63,18 @@ Remember that when you copy this repository for a new project you need to add th
 * `# wget -qO- https://deb.nodesource.com/setup_4.x | sudo bash -`
 * `# apt-get install --yes nodejs`
 
+Make sure npm version is 3.x
+* `$ npm -v`
+* `$ npm install -g npm@next` # only if version is lower than 3.x
+
 ### Main Project
 
 * `$ npm install`
 * `$ npm run dev`  # will run webpack with watch and compile code as it changes
 
 * `$ virtualenv -p /usr/bin/python3 virtualenv`
 * `$ source virtualenv/bin/activate`
+* `$ pip install -r py-requirements/base.txt`
```

Comment:
Aha! Didn't notice that :)

It must have been a coincidence then, I had some problems installing the dev.txt requirements, so I tried to install base.txt before, and everything went well, so I assumed that was the issue.
Thanks for the clarification!